extensions/rich_text_components/Link/directives/LinkDirective.js

```diff
@@ -51,6 +51,9 @@ oppia.directive('oppiaNoninteractiveLink', [
             if ($scope.text) {
               $scope.showUrlInTooltip = true;
             }
+            else {
+              $scope.text = $scope.url;
```

Comment:
Why are we doing this? Isn't it done in line 42?

Reply:
This is to avoid the case where text is left blank. In that case, the value will again be set to url.

Reply:
OK, I see. Thanks!